CMakeLists.txt

```diff
@@ -96,7 +97,11 @@ if(NBL_BUILD_ANDROID)
 	endif()
 	
 	string(LENGTH ${Java_JAVA_EXECUTABLE} Java_JAVA_EXECUTABLE_LENGTH)
-	set(Java_JAVA_EXE_BIN_LENGTH 13)
+	if(MSVC)
+		set(Java_JAVA_EXE_BIN_LENGTH 13)
+	else()
+		set(Java_JAVA_EXE_BIN_LENGTH 9)
```

Comment:
huh?

Reply:
That was borrowed from Danylo ("Some gcc fixes + Arek's cmake fixes" commit).

Reply:
@sadiuk || @AnastaZIuk explain?

Reply:
ok @AnastaZIuk explain please?

Reply:
on linux there is no `.exe` extension on executables, so we need to remove 4 characters

Reply:
ah ok